README.md

```diff
@@ -889,6 +889,38 @@ targets:
   - name: gem
 ```
 
+### AWS Lambda Layer
+
+The target will create a new public lambda layer in each available region with
+the extracted artifact from GitHub actions.
+
+**Environment**
+| Name | Description |
+| --------------------- | -------------------------------------------------------------------------- |
+| AWS_ACCESS_KEY | The access key of the AWS account to create and publish the layers. |
+| AWS_SECRET_ACCESS_KEY | The secret access key of the AWS account to create and publish the layers. |
```

Comment:
nit: You cleaned up the tables above, maybe you can do the same with the new ones?